modules/runners/lambdas/runners/src/aws/runners.ts

```diff
@@ -57,7 +60,24 @@ export async function listEC2Runners(filters: ListRunnerFilters | undefined = un
       ec2Filters.push({ Name: `tag:Owner`, Values: [filters.runnerOwner] });
     }
   }
-  const runningInstances = await ec2.describeInstances({ Filters: ec2Filters }).promise();
+
+  //  const runningInstances = [];
+
+  const runners: RunnerList[] = [];
+  let nextToken = undefined;
```

Comment:
Can you do this in typescript?

```suggestion
  let nextToken;
```